Yuneec_SDK_iOS/YNCSDKCamera.h

```diff
@@ -24,6 +24,37 @@ typedef void (^YNCCameraCompletion)(NSError *error);
  */
 typedef void (^YNCReceiveDataCompletionBlock)(YNCCameraSettings *receiveData, NSError *error);
 
+/**
+ Data type for the different color modes.
+ */
+typedef NS_ENUM (NSInteger, YNCCameraColorMode) {
+    /**
+     Neutral mode
+     */
+    YNCNEUTRAL = 0,
+    /**
+     Enhanced mode
+     */
+    YNCENHANCED,
```

Comment:
I wonder if `YNCENHANCED` is readable enough won't lead to conflicts.

I think I vote for:

```
typedef NS_ENUM (NSInteger, YNCCameraColorMode) {
   /**
    Neutral mode
    */
   YNCCameraColorModeNeutral = 0,
    ...
```

Reply:
Right. YNCCameraColorModeNeutral is better.